playbooks/provision.yml

```diff
@@ -119,6 +119,16 @@
       become_user: "{{ app_user }}"
       when: ansible_distribution_major_version >= '20'
 
+    - name: update new relic redis configuration # update redis port to monitor cache instance
+      replace:
+        dest: "/etc/newrelic-infra/integrations.d/redis-config.yml"
+        regexp: '^(\s*PORT:)\s6379$'
+        replace: '\1 6380'
```

Comment:
This should work, but it assumes although it's worth noting that if the port was set to anything else, it would silently skip over and do nothing. I'm not sure how to improve it though.